jekyll/_cci2/building-docker-images.md

```diff
@@ -10,22 +10,23 @@ version:
 - Server v2.x
 ---
 
-This document explains how to build Docker images for deployment elsewhere or further testing, and how to start services in a remote docker environment.
+This document explains how to build Docker images for deployment elsewhere or further testing, and how to start services in a remote docker environment when using the Docker execution environment.
```

Comment:
The first half of this sentence is a little confusing to read.

"This document explains how to build Docker images for deployment elsewhere or further testing, and how to start services in a remote docker environment when using the Docker execution environment."